SimG4CMS/HGCalTestBeam/test/waferGeom.C

```diff
@@ -0,0 +1,75 @@
+#include <algorithm>
```

Comment:
This is used for analyzing the MC data and used within ROOT. I made these functions and the analyzers of TB data use them and asked me to put it in the repository